### DISCUSSION TITLE
[@types/webextension-polyfill] Constant storage properties don't exist in Firefox

### DISCUSSION THREAD

**Discussion by MxtOUT** [DefinitelyTyped/DefinitelyTyped#66696] [Issues with a @types package]
As far as I can tell, `browser.storage.sync.QUOTA_BYTES_PER_ITEM` is `undefined` on Firefox, yet [the typings](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6dc5f15a7e02ba2c2664f8793866268220945851/types/webextension-polyfill/namespaces/storage.d.ts#L148) say its type should be `8192`.
While I have only tested this for said property, the same observation should be transferable to the other constants like `MAX_ITEMS` and `MAX_WRITE_OPERATIONS_PER_HOUR` (as well as to the other storage areas `local` and `managed`).

[a comment by an automated account is not shown]

**Comment by Lusito** [comment]
Thanks for the note. Not sure why Mozilla lists these properties in the schema files, but I've added a fix to not generate them: https://github.com/Lusito/webextension-polyfill-ts/commit/37957e2ba93c0d8e40ba17f34fb7807957c474a4

Will be added to the next version. I'll just check a couple of other things before I create a PR
